### PR TITLE
Docs: update do/do! indentation, etc.

### PR DESCRIPTION
**Pull request by toddlerer**
Changes proposed in this PR:
- `do` and `do!` are not indentation triggers, etc.
- Fixed a few other points.

@mtshiba

### CODE REVIEW

File: doc/EN/syntax/02_name.md [hunk 1]
```diff
@@ -113,12 +113,12 @@ You can delete an variable by using the `Del` function. All other variables that
 ```python,checker_ignore
 x = 1
 y = 2
-Z = 3
+z = 3
```
Comment (toddlerer) [inline doc/EN/syntax/02_name.md:116]: I think this is better in lower case for uniformity

File: doc/EN/terms.md [hunk 2]
```diff
@@ -123,6 +123,9 @@ Or the mutating operator.
 
 ### [Erg]
 
+Statically typed programming language published in 2022 and is the subject of this document. The developers pronounce it /ɝːɡ/. The name comes from the unit of energy in the CGS system of units.
```
Comment (toddlerer) [inline doc/EN/terms.md:126]: Is this close to how erg is pronounced in the U.S.?

File: doc/EN/syntax/02_name.md (hunk 1 above)
Comment (mtshiba) [reply doc/EN/syntax/02_name.md:116]: You're correct, since constants cannot be deleted.

File: doc/EN/terms.md (hunk 2 above)
Comment (toddlerer) [reply doc/EN/terms.md:126]: If it is pronounced the same as "エルグ" in Japanese, then `/ɝ́rug/` seems to be close. However, I am not familiar with phonetic alphabet, so I am not sure if this is appropriate.
How about [the link to google translate](https://translate.google.com/?sl=ja&tl=en&text=%E3%82%A8%E3%83%AB%E3%82%B0) in addition to the phonetic alphabet?

File: doc/JA/dev_guide/branches.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
 # ブランチの命名と運用方針
 
-[![badge](https://img.shields.io/endpoint.svg?url=https%3A%2F%2Fgezf7g7pd5.execute-api.ap-northeast-1.amazonaws.com%2Fdefault%2Fsource_up_to_date%3Fowner%3Derg-lang%26repos%3Derg%26ref%3Dmain%26path%3Ddoc/EN/dev_guide/branches.md%26commit_hash%3Da711efa99b325ba1012f6897e7b0e2bdb947d8a1)](https://gezf7g7pd5.execute-api.ap-northeast-1.amazonaws.com/default/source_up_to_date?owner=erg-lang&repos=erg&ref=main&path=doc/EN/dev_guide/branches.md&commit_hash=a711efa99b325ba1012f6897e7b0e2bdb947d8a1)
+[![badge](https://img.shields.io/endpoint.svg?url=https%3A%2F%2Fgezf7g7pd5.execute-api.ap-northeast-1.amazonaws.com%2Fdefault%2Fsource_up_to_date%3Fowner%3Derg-lang%26repos%3Derg%26ref%3Dmain%26path%3Ddoc/EN/dev_guide/branches.md%26commit_hash%3D14b0c449efc9e9da3e10a09c912a960ecfaf1c9d)](https://gezf7g7pd5.execute-api.ap-northeast-1.amazonaws.com/default/source_up_to_date?owner=erg-lang&repos=erg&ref=main&path=doc/EN/dev_guide/branches.md&commit_hash=14b0c449efc9e9da3e10a09c912a960ecfaf1c9d)
 
-* 基本的に開発は`main`ブランチ一本で行う(モノレポ開発)。どうしてもブランチを切らないと作業しにくい場合のみ`feature-*`ブランチか`issue-*`ブランチを作成する。
```
Comment (toddlerer) [inline doc/JA/dev_guide/branches.md:5]: I have removed this word because it is not appropriate in context

File: doc/EN/terms.md (hunk 2 above)
Comment (mtshiba) [reply doc/EN/terms.md:126]: This would be a good solution.

Comment (toddlerer) [reply doc/EN/terms.md]: OK, updated

**Comment by mtshiba** [comment]
Thanks!

**Pull request merged**